motor_control/throttle/button_control.ino

```diff
@@ -4,26 +4,62 @@ ButtonThrottle::ButtonThrottle(const int &pin, int timeToFull) : Throttle(pin, t
 {
   this->m_timeToFull = timeToFull;
   m_isPressed = false;
+  m_maxValue = 255;
+  m_speedOfCar = 0;
+  m_delaySpeed = 1;
 }
```

Comment:
The following variables are declared as private member variables of `ButtonThrottle` but I don't see them initialized in the constructor or anywhere else in the class? 
```
int m_pressStart;
int m_UnpressStart;
int m_delaySpeed;
int m_motor_value;
int m_decelerationFactor;
enum m_map_type
```